monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md

```diff
@@ -0,0 +1,22 @@
+# Faça Transações com Segurança, Use um Nodo Completo
```

Comment:
```suggestion
# Faça Transações com Segurança, Rode um Nó Completo
```
I would like to ask that you change all translations of "nodo" to "nó" as it is already widely used across the Monero Ecosystem ([Monerujo](https://github.com/m2049r/xmrwallet/search?q=n%C3%B3+remoto&unscoped_q=n%C3%B3+remoto), [Official Website](https://ww.getmonero.org/pt-br/resources/moneropedia/), [Official GUI Wallet](https://github.com/monero-project/monero-gui/search?q=n%C3%B3&unscoped_q=n%C3%B3) and so on.)